nodes/parachain/src/chain_spec.rs

```diff
@@ -146,6 +146,36 @@ pub fn staging_test_net(id: ParaId) -> Result<ChainSpec, String> {
 	))
 }
 
+pub fn peregrine_test_net(id: ParaId) -> Result<ChainSpec, String> {
+	let properties = get_properties("PKILT", 15, 38);
```

Comment:
What are these 15 and 38?

Reply:
And also PKILT?

Reply:
PKILT = peregrine-kilt

15 = 10^15 units are one PKILT

38 = The SS58 identifier

Yeah it's not that readable. But i would hope that's something for a different PR? 😅  We do it for all specs like that.